contrib/kind.sh

```diff
@@ -23,6 +23,10 @@ run_kubectl() {
 MASTER_COUNT=1
 K8S_VERSION=${K8S_VERSION:-v1.17.2}
 KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS:-false}
+export OVN_HYBRID_OVERLAY_ENABLE=${OVN_HYBRID_OVERLAY_ENABLE:-true}
```

Comment:
let's not default this to true

Reply:
modify the github actions to set the hybrid overlay to true when we run control plane jobs

Reply:
I started down the action path but then realized if anyone runs the e2e tests locally with KIND the tests will fail since HO is not enabled. I will hit you in Slack on this one.